CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+
+### Editing the Documentation
+
+Due to the rapid pace of development and limited resources, Firefly's documentation may not always be up-to-date or fully comprehensive. And here lies an opportunity for you: you can edit the documentation stored as markdown text files (`*.md`) in the [`docs` directory](https://github.com/Caltech-IPAC/firefly/tree/dev/docs) of Firefly and as docstrings in the `src` code files.
+
+You can see the Firefly JS docs at http://localhost:8080/firefly/docs/js/index.html if you have built them locally, as follows:
+```shell
+gradle firefly:buildJsDoc
+gradle firefly:buildAndDeploy
+```
+
+After editing the markdown or source files, build the docs again to check if your changes render correctly. Then you can submit your changes to us by making a patch as described in the next section.
+
+### Making a Patch (aka Code Contribution)
+
+Currently, Firefly tracks development priorities and tasks through Jira, which is internal to us, rather than through public GitHub issues. As a result, we don't have "good-first" or "easy" labeled issues to invite code contributions from new contributors. However, that shouldn't discourage you from contributing!  As a new user, you have a better perspective on where our documentation lacks, and you can edit it to make it better, as described in the previous section.
+
+If you've [set up Firefly locally](https://github.com/Caltech-IPAC/firefly/tree/dev?tab=readme-ov-file#setup) and have some familiarity with the codebase, we encourage you to contribute by adding features you'd find useful or fixing issues that affect your workflow. Your contributions will be invaluable to improve open-sourced Firefly for other users as well.
+
+To contribute your code, you'll need to create a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) from your fork of the Firefly repository. If this is your first time creating a pull request on GitHub, please refer to [this guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+
+When submitting a pull request, please provide a clear description of your changes, including any relevant testing instructions. Also, make sure your patch maintains code quality, is documented, and is testable.
+
+### Spreading the Word of Mouth
+
+If you find Firefly helpful, you can share it with your peers, colleagues, and anyone who can benefit from Firefly. If you've used Firefly in your research, please acknowledge us. By telling other people about how Firefly helped you, you'll help us in turn, extending Firefly's impact. And we would absolutely love it if you give us a shout-out on IPAC's social media: [LinkedIn](https://www.linkedin.com/showcase/ipac-at-caltech/) | [Twitter](https://x.com/caltechipac).
```

Comment:
Do we have any DOI for firefly that we can put for "acknowledge us"?

Reply:
I don't have an answer we will need to ask around.